src/client/Microsoft.Identity.Client/MsalServiceExceptionFactory.cs

```diff
@@ -83,6 +83,9 @@ private static bool IsThrottled(OAuth2ResponseBase oAuth2Response)
                 ex = new IntuneAppProtectionPolicyRequiredException(errorCode, subErrorCode)
                 {
                     Upn = msalTokenResponse.Upn,
+                    AuthorityUrl = msalTokenResponse.AuthorityUrl,
```

Comment:
Will these be avaiable on both iOS and Android?

Reply:
Authority is null in iOS. Others are available on both. Being null should not matter in iOS as it is not used.